cmd/svcat/output/output.go

```diff
@@ -53,6 +53,11 @@ func formatStatusFull(condition string, conditionStatus v1beta1.ConditionStatus,
 	return fmt.Sprintf("%s - %s @ %s", status, message, timestamp.UTC())
 }
 
+// WriteCreatedResourceName prints the name of a created resource
+func WriteCreatedResourceName(w io.Writer, resourceName string) {
```

Comment:
Is there a reason we don't have anything suitable for this in output/class.go?

Reply:
I don't understand the question/suggestion?

For other resources when we create them (like a binding), we have just printed out the details (basically the output of describe). We haven't had svcat create class resources yet so this is net new. Once we start allowing creating plans too, this function will be resued.

Reply:
No? We couldn't use something like WriteClass or WriteClassDetails?

Reply:
Oh! I didn't get that you are were suggesting that we output WriteClassDetails instead of just the name. 😊 Yeah that would be fine with me either way.